jerry-core/ecma/builtin-objects/ecma-builtin-helpers.h

```diff
@@ -130,45 +130,48 @@ extern ecma_completion_value_t ecma_date_get_primitive_value (ecma_value_t this_
 
 #endif /* !CONFIG_ECMA_COMPACT_PROFILE_DISABLE_DATE_BUILTIN */
 
-typedef struct
-{
-  void **block_start_p;
-  void **block_end_p;
-  void **current_p;
-} list_ctx_t;
+/* ecma-builtin-helper-json.cpp */
 
+/**
+ * Context for JSON.stringify()
+ */
 typedef struct
 {
-  list_ctx_t property_list;
-  list_ctx_t occurence_stack;
+  /** Collection for property keys. */
+  ecma_collection_header_t *property_list_p;
+
+  /** Collection for traversing objects. */
+  ecma_collection_header_t *occurence_stack_p;
 
+  /** The actual indentation text. */
   ecma_string_t *indent_str_p;
+
+  /** The indentation text. */
   ecma_string_t *gap_str_p;
+
+  /** The replacer function. */
   ecma_object_t *replacer_function_p;
 } stringify_context_t;
```

Comment:
Could you, please, also add `ecma_json_` or `ecma_` prefix to the structure name?